src/client/app/containers/app.ts

```diff
@@ -18,8 +18,9 @@ import { Strings, environment } from '../helpers';
                     <command icon="OpenPaneMirrored" title="${Strings.runSideBySide}" (click)="runSideBySide()"></command>
                 </command>
                 <command [hidden]="isEmpty" icon="Share" [async]="sharing$|async" title="${Strings.share}">
-                    <command icon="PageCheckedin" title="${Strings.shareMenuPublic}" (click)="shareGist(true)"></command>
-                    <command icon="ProtectedDocument" title="${Strings.shareMenuPrivate}" (click)="shareGist(false)"></command>
+                    <command *ngIf="snippet?.isOwned" icon="Save" title="${Strings.updateMenu}" (click)="shareGist(null, true)"></command>
```

Comment:
Haven't been able to find a way to leave inline comments in the Angular template. Perhaps HTML comments, or there's something I've missed? Would welcome any suggestions.